src/usageReport.ts

```diff
@@ -0,0 +1,55 @@
+import { v4 as uuidv4 } from 'uuid'
```

Comment:
We use `self.crypto.randomUUID` in others places in k6 studio, I don't think a separate module is needed

Reply:
That's on me!
I didn't notice :)